docs/lib/restapi/fragments/android/getting-started/11_amplifyInit.md

```diff
@@ -0,0 +1,49 @@
+To start provisioning api resources in the backend, go to your project directory and **execute the command**:
+
+```bash
+amplify add api
+```
+
+Enter the following when prompted:
+```console
+? Please select from one of the below mentioned services: 
+    `REST`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `api`
+? Provide a path (e.g., /book/{isbn}): 
+    `/todo`
+? Choose a Lambda source 
+    `Create a new Lambda function`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `todo`
+? Provide the AWS Lambda function name: 
+    `todo`
+? Choose the function runtime that you want to use: 
+    `NodeJS`
+? Choose the function template that you want to use: 
+    `Serverless ExpressJS function (Integration with API Gateway)`
+? Do you want to access other resources created in this project from your Lambda function? 
+    `No`
+? Do you want to invoke this function on a recurring schedule? 
+    `No`
+? Do you want to edit the local lambda function now?
+    `No`
+? Restrict API access 
+    `No`
```

Comment:
should we link to https://github.com/aws-amplify/amplify-android/issues/537 as a TODO comment in this file to remember to consolidate the provisioning back to using auth, or perhaps we should just disable auth all together for ios as well for an easier run-through?

Reply:
I'm not opposed to either.